nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-api/src/main/java/org/apache/nifi/oauth2/AccessToken.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.nifi.oauth2;
+
+public class AccessToken {
+    private String accessToken;
+    private String refreshToken;
+    private String tokenType;
+    private Integer expires;
+    private String scope;
+
+    private Long fetchTime;
+
+    public AccessToken() {}
```

Comment:
Typically a default constructor is not explicitly defined

Reply:
That was put there to appease Sonarqube for our client. I can remove it.

Reply:
ahh just leave it then. That is a petty syntax thing anyway.